<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1099</width>
    <height>757</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>480</y>
      <width>761</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabCanvas">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>761</width>
      <height>481</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1099</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="meView">
    <property name="title">
     <string>视图</string>
    </property>
    <addaction name="acFile"/>
    <addaction name="acDevLib"/>
    <addaction name="acCanvas"/>
   </widget>
   <widget class="QMenu" name="meFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="acNew"/>
    <addaction name="acOpen"/>
    <addaction name="acSave"/>
   </widget>
   <addaction name="meFile"/>
   <addaction name="meView"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="acNew"/>
   <addaction name="acOpen"/>
   <addaction name="acSave"/>
   <addaction name="separator"/>
   <addaction name="acTopology"/>
   <addaction name="acAnalysis"/>
   <addaction name="separator"/>
   <addaction name="acRun"/>
   <addaction name="acPause"/>
   <addaction name="acStop"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockProject">
   <property name="minimumSize">
    <size>
     <width>25</width>
     <height>50</height>
    </size>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>项目资源管理器</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="widProject"/>
  </widget>
  <widget class="QDockWidget" name="dockLibrary">
   <property name="minimumSize">
    <size>
     <width>25</width>
     <height>50</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>器件库</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="widLibrary"/>
  </widget>
  <action name="acNew">
   <property name="icon">
    <iconset>
     <normaloff>../../Pictures/Saved Pictures/图片2.png</normaloff>../../Pictures/Saved Pictures/图片2.png</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
   <property name="iconText">
    <string>新建</string>
   </property>
   <property name="toolTip">
    <string>新建文件</string>
   </property>
  </action>
  <action name="acOpen">
   <property name="icon">
    <iconset>
     <normaloff>../../Pictures/Saved Pictures/打开 (6).png</normaloff>../../Pictures/Saved Pictures/打开 (6).png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开文件</string>
   </property>
  </action>
  <action name="acSave">
   <property name="icon">
    <iconset>
     <normaloff>../../Pictures/Saved Pictures/保存 (2).png</normaloff>../../Pictures/Saved Pictures/保存 (2).png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存文件</string>
   </property>
  </action>
  <action name="acTopology">
   <property name="icon">
    <iconset>
     <normaloff>../../Pictures/Saved Pictures/网络拓扑.png</normaloff>../../Pictures/Saved Pictures/网络拓扑.png</iconset>
   </property>
   <property name="text">
    <string>网络拓扑</string>
   </property>
   <property name="toolTip">
    <string>网络拓扑规划</string>
   </property>
  </action>
  <action name="acAnalysis">
   <property name="icon">
    <iconset>
     <normaloff>../../Pictures/Saved Pictures/性能分析.png</normaloff>../../Pictures/Saved Pictures/性能分析.png</iconset>
   </property>
   <property name="text">
    <string>性能分析</string>
   </property>
   <property name="toolTip">
    <string>性能分析报告</string>
   </property>
  </action>
  <action name="acRun">
   <property name="icon">
    <iconset>
     <normaloff>../../Pictures/Saved Pictures/1开始.png</normaloff>../../Pictures/Saved Pictures/1开始.png</iconset>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
   <property name="toolTip">
    <string>运行仿真</string>
   </property>
  </action>
  <action name="acPause">
   <property name="icon">
    <iconset>
     <normaloff>../../Pictures/Saved Pictures/1暂停.png</normaloff>../../Pictures/Saved Pictures/1暂停.png</iconset>
   </property>
   <property name="text">
    <string>暂停</string>
   </property>
   <property name="toolTip">
    <string>暂停仿真</string>
   </property>
  </action>
  <action name="acStop">
   <property name="icon">
    <iconset>
     <normaloff>../../Pictures/Saved Pictures/图片4.png</normaloff>../../Pictures/Saved Pictures/图片4.png</iconset>
   </property>
   <property name="text">
    <string>终止</string>
   </property>
   <property name="toolTip">
    <string>终止仿真</string>
   </property>
  </action>
  <action name="acDevLib">
   <property name="text">
    <string>器件库</string>
   </property>
  </action>
  <action name="acCanvas">
   <property name="text">
    <string>画布</string>
   </property>
  </action>
  <action name="acFile">
   <property name="text">
    <string>项目资源管理器</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
